.github/workflows/ci.yml

```diff
@@ -7,6 +7,17 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
This will need to be in `deploy-docs-prod.yml`. The `ci.yml` deploys to `dev` already on merges to `main`.